src/components/Button/Button.stories.tsx

```diff
@@ -13,73 +11,79 @@ export default {
   title: 'Components|Button',
 };
 
-const onPress = () => { };
+const onPress = () => {};
```

Comment:
Sugestão: adicionar `Function.prototype()` no conteúdo do bloco desta _arrow function_.
_Arrow functions_ com bloco vazio podem ser considerados _code smell_.

Reply:
e Function.prototype() não? uma função vazia é sempre uma função vazia

Reply:
isso é uma duvida real, se tuver alguma literatura ai pra me ajudar eu adoraria :)

Reply:
O `eslint`, dependendo da configuração, reclama se encontrar uma _arrow function_ com bloco vazio.
Eu passei por isso em um projeto, e fiquei pensando em como chamar uma função noop para não deixá-lo vazio.

Então eu caí nesta "literatura" do StackOverflow haha:
https://stackoverflow.com/questions/21634886/what-is-the-javascript-convention-for-no-operation

Reply:
mas aí pela "literatura" eu entendo que: 

> "The most concise and performant noop is an **empty arrow function**: `()=>{}`."